packages/manager/modules/dbaas-logs/src/logs/list/logs-list.html

```diff
@@ -12,8 +12,7 @@
                 <oui-datagrid-topbar>
                     <a
                         class="oui-button oui-button_secondary"
-                        target="_blank"
-                        data-ng-href="{{ctrl.orderURL}}"
+                        data-ui-sref="{{ctrl.orderURL}}"
```

Comment:
We try to avoid `ui-sref` in views, we prefer using a ng-href and inject everything related to ui router in the state declaration especially in `resolve`

Reply:
thanks for the tips, it's done ^^